app/Providers/RouteServiceProvider.php

```diff
@@ -40,8 +40,7 @@ public function boot()
 
         $this->routes(function () {
 
-            Route::prefix('api')
-                ->middleware('api')
+            Route::middleware('api')
                 ->namespace($this->namespace)
                 ->group(base_path('routes/api.php'));
 
```

Comment:
Also need to remove the SPA group, the spa route files and the SPA controller here I believe